opm/autodiff/FlowMainEbos.hpp

```diff
@@ -410,14 +411,27 @@ namespace Opm
 
         void setupEbosSimulator()
         {
-            std::string progName("flow_ebos");
-            std::string deckFile("--ecl-deck-file-name=");
-            deckFile += param_.get<std::string>("deck_filename");
-            char* ptr[2];
-            ptr[ 0 ] = const_cast< char * > (progName.c_str());
-            ptr[ 1 ] = const_cast< char * > (deckFile.c_str());
+            std::vector<const char*> argv;
+
+            argv.push_back("flow_ebos");
+
+            std::string deckFileParam("--ecl-deck-file-name=");
+            deckFileParam += param_.get<std::string>("deck_filename");
+            argv.push_back(deckFileParam.c_str());
+
+#if defined(_OPENMP)
+            std::string numThreadsParam("--threads-per-process=");
+            int numThreads = 1;
+#pragma omp parallel
```

Comment:
omp_get_max_threads() ? or maybe omp_get_thread_limit() ?

Reply:
Maybe, I just copy and pasted [this](https://github.com/OPM/opm-simulators/blob/master/opm/autodiff/FlowMainEbos.hpp#L172). shall I change both to `omp_get_max_threads()`?

Reply:
that's the equivalent, ie number of threads used for next parallel section. omp_get_thread_limit() is the amount available on start, ie, that is not affected by omp_set_num_threads() calls.

Reply:
thanks for the info. update pushed.